common/djangoapps/static_replace/__init__.py

```diff
@@ -148,7 +148,7 @@ def replace(__, prefix, quote, rest):
     )
 
 
-def replace_static_urls(text, data_directory=None, course_id=None, static_asset_path=''):
+def replace_static_urls(text, data_directory=None, course_id=None, static_asset_path='', static_paths=None):
```

Comment:
@pomegranited One quick request: could you rename this arg to `static_paths_out` or `replacement_list_out`? I think that would be more clear. Then please squash and ping me for merge.

Reply:
Sure @bradenmacdonald -- see f108b65